src/content/docs/browser/new-relic-browser/page-load-timing-resources/pageviewtiming-async-or-dynamic-page-details.mdx

```diff
@@ -134,7 +132,7 @@ The `BrowserInteraction` and `PageView` events end their reporting when they rec
       </td>
 
       <td>
-        [Interaction to Next Paint (INP)](https://web.dev/inp/) is available with [agent v1227 or higher](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1227). INP is a newer metric for measuring [runtime responsiveness](https://web.dev/user-centric-performance-metrics/#types-of-metrics) and user-perceived performance. It measures the largest latency between user interactions and page response or repaints. This is an experimental but identified-as-significant metric added in [Web Vitals](https://github.com/GoogleChrome/web-vitals) v3.
+        [Interaction to next paint (INP)](https://web.dev/inp/) is available with [agent v1227 or higher](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1227). INP is a newer metric for measuring [runtime responsiveness](https://web.dev/user-centric-performance-metrics/#types-of-metrics) and user-perceived performance. It measures the largest latency between user interactions and page response or repaints. This is an experimental but identified-as-significant metric added in [Web Vitals](https://github.com/GoogleChrome/web-vitals) v3.
```

Comment:
assuming Web Vitals is stylized like this because it's a chrome extension of some kind, but calling it out just in case